### DISCUSSION TITLE
不能方便的复制字段,然后不能选中一条或多条数据后直接复制该记录的insert或者update语句

### DISCUSSION THREAD

**Discussion by yhh1314** [OtterMind/Chat2DB#679] [General]
不能方便的复制字段,然后不能选中一条或多条数据后直接复制该记录的insert或者update语句,最后就是希望出一个快捷生成测试数据的功能,像navicat看齐

**Comment by liuyuzhuo** [comment]
确实，dbever的选择某几行，高级复制为insert语句的功能太实用了，希望改进下，加上这个功能。

**Comment by xyombo** [comment]
具体一点，我想要 DataGrip 类似的功能，选择若干行后，能够复制出 `insert` `update` `csv` 格式的数据
<img width="243" alt="image" src="https://github.com/chat2db/Chat2DB/assets/40785594/3bf0738a-a85d-4d0a-a8a9-d1c0634e5391">

**Comment by shanhexi** [comment]
目前支持了单条数据的复制insert或者update语句，后续1-2周内支持 多条复制

**Comment by openai0229** [comment]
现在支持了哦
